org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -528,4 +531,130 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 		#{"**/*.xtend","**/*.mwe2"}
 	}
 	
+	/**
+	 * @since 2.11
+	 */
+	def private getWfLaunchConfigFile() {
```

Comment:
For me it's OK to merge. By the way, I cannot check it now: the @since annotations are required by the API tooling for public and protected methods; For private methods they are not required, but do you get any error from the API tooling if you have @since annotations?